theme_yes/i18n/theme_yes.pot

```diff
@@ -119,7 +119,12 @@ msgstr ""
 
 #. module: theme_yes
 #: model_terms:theme.ir.ui.view,arch:theme_yes.s_masonry_block_default_template
-msgid "Check out our list of favorite venues."
+msgid "Check out our list of favorite caterers."
+msgstr ""
+
+#. module: theme_yes
+#: model_terms:theme.ir.ui.view,arch:theme_yes.s_masonry_block_default_template
+msgid "Find a Caterer"
```

Comment:
Could you please adapt `.pot` so it is the same as the one generated through the UI (settings>translations>export)?
Those should be ordered alphabetically, just generate the .pot from Odoo and check in a text compare tool where should be placed the string you added.
Otherwise, it will keep introducing fake diff whenever someone will generate the .pot for their own modifications.

Reply:
Hello, the `.pot` file was updated based on generated through the UI.